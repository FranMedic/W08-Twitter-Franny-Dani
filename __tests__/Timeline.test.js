import { render, screen } from "@testing-library/react";
import Timeline from "../pages/timeline/index";

describe("Given a timeline component", () => {
  describe("When it's rendered", () => {
    test("Then it should shows tuits list", () => {
      render(<Timeline />);

      const timeline = screen.getByRole("heading", { name: /Timeline/i });
      expect(timeline).toHaveTextContent("Timeline");
    });
  });
});
