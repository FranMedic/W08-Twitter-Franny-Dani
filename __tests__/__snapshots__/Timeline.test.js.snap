// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Given a timeline component When it recives and object then it should render a tuit with the tuit text inside 1`] = `
<div
  className="tuitContainer"
>
  <h2
    className="tuitTitle"
  >
    Timeline
  </h2>
  <ul
    className="tuitlist"
  >
    <li
      className="tuit"
    >
      <a
        href="/timeline/61969108c5fead84ba7b8a32"
        onClick={[Function]}
        onMouseEnter={[Function]}
      >
        patata

      </a>
      <p>
        0
         Likes
      </p>
      <time
        dateTime="2021-11-18T17:44:40.529Z"
      >
        hace 14 horas
      </time>
      <button
        className="like"
        onClick={[Function]}
      >
        Like
      </button>
    </li>
  </ul>
</div>
`;
